src/IconButton/index.jsx

```diff
@@ -59,7 +53,7 @@ const IconButton = React.forwardRef(({
 });
 
 IconButton.defaultProps = {
-  iconAs: undefined,
+  iconAs: Icon,
```

Comment:
Can you please also update examples on docs site? Since `Icon` is the default value now, we no longer need to explicitly pass `iconsAs={Icon}` and I think we do it in all of the examples currently.

Reply:
Yes, good catch
Thanks 👍